src/js/components/FormField/stories/FieldStatesCustom.js

```diff
@@ -0,0 +1,138 @@
+      },
+      label: {
+        color: 'status-disabled',
+      },
+    },
+    error: {
+      background: {
+        color: { light: '#FF404033', dark: '#FF40404D' },
+      },
+      size: 'xsmall',
+      color: 'text-weak',
+      margin: {
+        start: 'none',
+      },
+    },
+    help: {
+      size: 'xsmall',
+      color: 'text-weak',
+      margin: {
+        start: 'none',
+        bottom: 'xsmall',
+      },
+    },
+    info: {
+      size: 'xsmall',
+      color: 'text-weak',
+      margin: {
+        start: 'none',
+      },
+    },
+    label: {
+      size: 'xsmall',
+      color: 'text-weak',
+      margin: {
+        horizontal: 'none',
+      },
+    },
+    round: '4px',
+  },
+});
+
+const CustomFormFieldStates = () => {
+  const inputRef = useRef();
+
+  useEffect(() => {
+    inputRef.current.focus();
+  }, []);
+
+  return (
+    <Grommet theme={customTheme}>
+      <Box align="center" pad="large">
+        <Form>
+          <Box border gap="medium" pad="large" width="medium">
+            <FormField htmlFor="enabled-id" name="enabled" label="Default">
+              <TextInput
+                id="enabled-id"
+                name="enabled"
+                placeholder="Enter a username"
+              />
+            </FormField>
+            <FormField htmlFor="focus-id" name="focus" label="Focus State">
+              <TextInput
+                id="focus-id"
+                name="focus"
+                placeholder="Enter a username"
+                ref={inputRef}
+              />
+            </FormField>
+            <FormField
+              htmlFor="info-id"
+              name="info-demo"
+              label="Info State"
+              info="Unique name. No spaces. May include '-' as a separator."
+            >
+              <TextInput
+                id="info-id"
+                name="info-demo"
+                placeholder="Enter a username"
+                value="fluffyKi"
+              />
+            </FormField>
+            <FormField
+              htmlFor="error-id"
+              name="error-demo"
+              label="Error State"
+              error="It looks like that username is already taken. Bummer."
+            >
+              <TextInput
+                id="error-id"
+                name="error-demo"
+                placeholder="Enter a username"
+                value="fluffyKitty123"
+              />
+            </FormField>
+            <FormField
+              htmlFor="disabled-id"
+              name="disabled"
+              label="Disabled State"
+              disabled
+            >
+              <TextInput
+                id="disabled-id"
+                name="disabled"
+                placeholder="Enter a username"
+                disabled
+              />
+            </FormField>
+          </Box>
+        </Form>
+      </Box>
+    </Grommet>
+  );
+};
+
+storiesOf('FormField', module).add('Field States Custom', () => (
```

Comment:
Since you added a Form to the example, I think it will be better to put it on the Form module, our goal is to clean up the FormField stories.
```suggestion
storiesOf('Form', module).add('Field States Custom', () => (
```
And also move the file to Form stories, same for the other stories.

Reply:
Addressed. Done.